packages/contracts-bedrock/README.md

```diff
@@ -58,3 +58,47 @@ npx hardhat deploy --network <network-name>
 In the `hardhat.config.ts`, there is a `deployConfigSpec` field that validates that the types
 are correct, be sure to export an object in the `deploy-config/<network-name>.ts` file that
 has a key for each property in the `deployConfigSpec`.
+
+## Standards and Conventions
+
+### Style
+
+#### Comments
+
+We use [Seaport](https://github.com/ProjectOpenSea/seaport/blob/main/contracts/Seaport.sol)-style comments with some minor modifications.
+Some basic rules:
+* Always use `@notice` since it has the same general effect as `@dev` but avoids confusion about when to use one over the other.
+* Include a newline between `@notice` and the first `@param`.
+* Include a newline between `@param` and the first `@return`.
+* Use a line-length of 100 characters.
+
+We also have the following custom tags:
+* `@custom:proxied`: Add to a contract whenever it's meant to live behind a proxy.
+* `@custom:legacy`: Add to an event or function when it only exists for legacy support.
+
+#### Errors
+
+* Use `require` statements when making simple assertions.
+* Use `revert` if throwing an error where an assertion is not being made (no custom errors).
+* Error strings MUST have the format `"{ContractName}: {message}"` where `message` is a lower case string.
+
+#### Function Parameters
+
+* Function parameters should be prefixed with an underscore.
+
+#### Event Parameters
+
+* Event parameters should NOT be prefixed with an underscore.
+
+### Proxy by Default
+
+Optimism follows a philosophy of fast and frequent iteration.
+In an effort to increase the speed at which we can iterate, all contracts should be assumed to live behind proxies (except in certain special circumstances).
```

Comment:
This statement feels too strong to me, and seems to imply that we play 'fast and loose'. In practice, because we have a policy of auditing everything we deploy, we don't actually perform upgrades on a regular basis.

IMO the primary motivation for upgradability is for security reasons (ie. patching bugs). A close second is so that we can optimize and improve the contracts overtime.

If we disagree, then I propose we sidestep providing reasons here, and just skip to the line below.